### ISSUE TITLE
[FEATURE] macOS support

### ISSUE THREAD

**Issue by MatadorProBr** [OhMyGuus/BetterCrewLink#265]
### Pre-Flight Checklist

- [X] I am not asking a question => use the [Discord](https://discord.gg/qDqTzvj4SH) if you are
- [X] I have checked that someone else has not reported this using the [search bar](https://github.com/OhMyGuus/BetterCrewLink/issues?q=is%3Aissue)
- [X] My feature request is not one of the **commonly suggested features**: iOS app (it's already being development)

### Is your feature request related to a problem? Please describe.

No. It's kind of annoying for mac users to have to use the web version because they need a PC user using the BCL app, so I think it would be a good idea to port the BCL app from Windows & Linux to macOS, like the #26 and #70 but instead of Linux, it's on macOS

### Describe the solution you'd like

Make the BetterCrewLink app from Windows & Linux works on a macOS.

### Describe alternatives you've considered

N/A

**Comment by OhMyGuus** [comment]
Pretty sure there is no Among Us on MacOS so no way to support something that doesn't exist..

**Comment by MatadorProBr** [comment]
> Pretty sure there is no Among Us on MacOS so no way to support something that doesn't exist..

Yes but exist PlayOnMac, CrossOver and others things that make Among Us on Steam, itch.io, Epic run on a macOS

**Comment by kristibektashi** [comment]
> Pretty sure there is no Among Us on MacOS so no way to support something that doesn't exist..

There is no Among Us for Linux either, it runs via Proton. Also, as the above comment said, Crossover, PlayOnMac and even Wine (via Homebrew or Macports) exist and can allow Among Us to run on MacOS. (There is also the iPad version available on the App Store but that still uses mobile controls so it's kinda janky)

**Comment by ezamelczyk** [comment]
It's literally on mac app store.

**Comment by filipon12** [comment]
@ezamelczyk but it is the iPad version  

<img width="3164" height="1906" alt="Image" src="https://github.com/user-attachments/assets/804159a9-77a7-4ee0-a931-d377ff2c5cef" />


It literally says "Designed for iPad"

**Comment by OhMyGuus** [comment]
We could make the mobile version an desktop app. But it requires atleast one player on pc for it
